.github/workflows/maven.yml

```diff
@@ -31,6 +31,15 @@ jobs:
         maven-version: 3.9.6
     - name: Build with Maven
       run: mvn -B package --file pom.xml -DunofficialSuffix=
+    - name: Arrange P2 Repsository
+      run: |
+        cd com.wamas.ide.launching.update/target/repository/
+        mv plugins/* .
+        mv features/* .
+        sed -i 's/\/plugins\//\//g' artifacts.xml
```

Comment:
Not using `/` as the separator for sed command allows to avoid escaping in the regex, e.g:
`sed -i 's,/plugins/,/,g' ...`